### ISSUE TITLE
Normalize data and add final layer activation 

### ISSUE THREAD

**Issue by chauhankaranraj** [thakkarV/DeepFriedGIFs#2]
Transform (normalize) input data before feeding it to encoder. Add activation to output logits of final layer of decoder

**Comment by thakkarV** [comment]
transform support added to argument parser and dataset reader. train.py can now import a transform and pass it to dataset.
@chauhankaranraj can you take on the activation to network part?

**Comment by thakkarV** [comment]
fixed in 70d588a2f8bd192d702f3a3e1b40a8a3b2b7ec13
